util/topgen/intermodule.py

```diff
@@ -703,19 +726,19 @@ def check_intermodule(topcfg: Dict, prefix: str) -> int:
         if error != 0:
             # Skip the check
             continue
-        rsps_width = 0
-        for rsp in rsps:
-            rsp_m, rsp_s, rsp_i = filter_index(rsp)
-            rsp_struct = find_intermodule_signal(
-                topcfg["inter_signal"]["signals"], rsp_m, rsp_s)
-            # Update total responses width
-            rsps_width += rsp_struct["width"]
-
-        if req_struct["width"] != rsps_width:
-            log.error(
-                "Request {} width is not matched with total responses width {}"
-                .format(req_struct["width"], rsps_width))
-            error += 1
+        #rsps_width = 0
+        #for rsp in rsps:
+        #    rsp_m, rsp_s, rsp_i = filter_index(rsp)
+        #    rsp_struct = find_intermodule_signal(
+        #        topcfg["inter_signal"]["signals"], rsp_m, rsp_s)
+        #    # Update total responses width
+        #    rsps_width += rsp_struct["width"]
+        #
+        #if req_struct["width"] != rsps_width:
+        #    log.error(
+        #        "Request {} width is not matched with total responses width {}"
+        #        .format(req_struct["width"], rsps_width))
+        #    error += 1
```

Comment:
yeah sorry, i did eventually remooe this (in yet another PR), but this was actually something i was trading comments on with @eunchan because it was causing topgen errors that I think are bogus.